bin/git-mr

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if test "$1" = "clean"; then
+  git for-each-ref refs/heads/mr/* --format='%(refname)' | while read ref; do
+    git branch -D ${ref#refs/heads/}
+  done
+  exit 0
+elif [[ $1 =~ ^(https?://[^/]+/(.+))/merge_requests/([0-9]+).*$ ]]; then
+  remote=${BASH_REMATCH[1]}.git
+  id=${BASH_REMATCH[3]}
+  branch=mr/$id
+else
+  test -z $1 && echo "mr number required." 1>&2 && exit 1
+  id=$1
+  remote=${2:-origin}
+  branch=mr/$id
+fi
+git fetch -fu $remote refs/merge-requests/$id/head:$branch && \
+git checkout $branch && \
+git config --local --replace branch.$branch.merge refs/pull/$id/head && \
```

Comment:
It should be `refs/merge-requests/$id/head` instead of `refs/pull/$id/head` here.